tests/check_executables_have_shebangs_test.py

```diff
@@ -33,3 +44,67 @@ def test_bad_shebang(content, tmpdir, capsys):
     assert main((path.strpath,)) == 1
     _, stderr = capsys.readouterr()
     assert stderr.startswith(f'{path}: marked executable but')
+
+
+def test_check_git_filemode_passing(tmpdir):
+    with tmpdir.as_cwd():
+        cmd_output('git', 'init', '.')
+
+        f = tmpdir.join('f')
+        f.write('#!/usr/bin/env bash')
+        f_path = str(f)
```

Comment:
The other tests use `.strpath`, is one preferred over the other?

Reply:
apparently `strpath` is not documented despite being a public attribute so I've mostly switched to `str(...)` now that I don't care about python2

Reply:
TIL